Laravel.gitignore

```diff
@@ -2,3 +2,13 @@
 .env.*.php
 .env.php
 .env
+/node_modules
+/.idea/
```

Comment:
We've bumped up against some rule updates to the JetBrains IDEs - see #1475 for the details.